tests/test_transforms/test_math/test_limit_transform.py

```diff
@@ -0,0 +1,146 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from etna.datasets import TSDataset
+from etna.metrics import MSE
+from etna.models import NaiveModel
+from etna.pipeline import Pipeline
+from etna.transforms.math import LimitTransform
+
+
+@pytest.fixture
+def ts_check_pipeline_with_limit_transform(random_seed) -> TSDataset:
+    periods = 100
+    df1 = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
+    df1["segment"] = ["segment_1"] * periods
+    df1["target"] = np.arange(periods)
+    df1.loc[5, "target"] = np.NaN
+
+    df2 = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
+    df2["segment"] = ["segment_2"] * periods
+    df2["target"] = np.random.uniform(-15, 5, size=periods)
+    df2.loc[10, "target"] = np.NaN
+
+    df = pd.concat((df1, df2))
+    df = TSDataset.to_dataset(df)
+    tsds = TSDataset(df, freq="D")
+    return tsds
+
+
+@pytest.fixture
+def ts_check_limit_transform(random_seed) -> TSDataset:
+    periods = 4
+    df = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
+    df["segment"] = ["segment_1"] * periods
+    df["target"] = np.array([1, 2, 3, 4])
+    df["target_transformed_1"] = np.array([1, 2, 3, 4])
+    df["target_transformed_2"] = np.array(
+        [1.3862943611448906, 1.0986122887014431, 0.6931471806099453, 1.000000082690371e-10]
+    )
+    df["target_transformed_3"] = np.array(
```

Comment:
Good practice is to chose the initial values that gives the simple expected answer like 1, 10, 100, 1000 gives 0, 1, 2, 3 -> here you just copy the result from applying the formulas. Try to come up with reasonable values that you can really evaluate yourself. Values for non_target column might be the same(and the answers too)

Reply:
in addition to one simple number in logarithm you have two borders that you apply to each number in array + tol
I can't come up by myself and Wolfram too

Reply:
Then it is sth wrong with your code if it is hard to test how it works

Reply:
But here it may be really hard to come up with good test case